configure.sh

```diff
@@ -140,6 +126,24 @@ export HTML_DIRECTORY=$ROOT_WEB_DIRECTORY
 mkdir /var/www/html/competitions
 chown vagrant:www-data /var/www/html/competitions
 
+# Install SimpleBook
+echo "INSTALL SIMPLEBOOK SERVER"
+cd /home/vagrant/torque-sites/base/simplebook/ansible
+envsubst < inv/local/group_vars/all.tmpl > inv/local/group_vars/all
+ansible-playbook simplebook.yml -i inv/local
+
+# Overwrite SimpleBook installation with mounted folder
+if [[ -L $SIMPLEBOOK_INSTALL_DIRECTORY ]]; then
+  echo "WARNING: The SIMPLEBOOK_INSTALL_DIRECTORY is already a symlink, skipping mount linking step."
+else
+	rm -fr $SIMPLEBOOK_INSTALL_DIRECTORY
+	ln -s /home/vagrant/SimpleBook $SIMPLEBOOK_INSTALL_DIRECTORY
+	cd $SIMPLEBOOK_INSTALL_DIRECTORY/services/api
+	pipenv install
+	cd $SIMPLEBOOK_INSTALL_DIRECTORY/services/api/mw2pdf
+	yarn install
+fi
+
```

Comment:
> It also replaces the actual code base of the ansible-installed SimpleBook with our mounted copy so that we can actually develop the project in our development environment.

We are doing this *here*?